src/components/buttons/DeleteButton.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { PropTypes } from 'react';
-
 import IconButton from 'material-ui/IconButton';
 import TrashCan from 'material-ui/svg-icons/action/delete';
 import ReactTooltip from 'react-tooltip';
+import { branch, renderNothing } from 'recompose';
```

Comment:
I know that's an established pattern, but it's not one that has benefit over this method. Recompose helps make reasoning about components easier, it's an essential for FP in React, and it's more readable vs. the ternary operator.